satpy/readers/ghrsst_l2.py

```diff
@@ -24,21 +27,18 @@
 from satpy.readers.file_handlers import BaseFileHandler
 
 
-class SLSTRL2FileHandler(BaseFileHandler):
-    """File handler for Sentinel-3 SSL L2 netCDF files."""
+class GHRSSTL2FileHandler(BaseFileHandler):
+    """File handler for GHRSST L2 netCDF files."""
 
     def __init__(self, filename, filename_info, filetype_info, engine=None):
-        """Initialize the file handler for Sentinel-3 SSL L2 netCDF data."""
-        super(SLSTRL2FileHandler, self).__init__(filename, filename_info, filetype_info)
+        """Initialize the file handler for GHRSST L2 netCDF data."""
+        super(GHRSSTL2FileHandler, self).__init__(filename, filename_info, filetype_info)
```

Comment:
Python 3 doesn't need this.
```suggestion
        super().__init__(filename, filename_info, filetype_info)
```

Reply:
Ah, thanks!